### DISCUSSION TITLE
Found this code, think it might help.

### DISCUSSION THREAD

**Discussion by Polygons1** [Pumpkin-MC/Pumpkin#232] [General]
https://github.com/dmulloy2/ProtocolLib/blob/master/src/main/java/com/comphenix/protocol/PacketType.java

**Comment by Snowiiii** [comment]
Mhh, Not really. The Link you give is are just Packet Types implemented by Protocol Lib, Nothing we can't find ourselfs. 

 Wiki.vg is good, but since yesterday i started to lose trust in wiki.vg. The best source for Packet implementations is still Vanilla code
